worker/src/main/scala/org/apache/celeborn/service/deploy/worker/Worker.scala

```diff
@@ -151,8 +150,20 @@ private[celeborn] class Worker(
   storageManager.disksSnapshot().foreach { case diskInfo =>
     diskInfos.put(diskInfo.mountPoint, diskInfo)
   }
+
+  // need to ensure storageManager has recovered fileinfos data before retrieve consumption
+  val userResourceConsumption = storageManager.userResourceConsumptionSnapshot().asJava
```

Comment:
Just for double check, here it will be executed after the data is restored?

Reply:
> Just for double check, here it will be executed after the data is restored?

if you write this code after initializing StorageManager, here will execute after initializing StorageManager